### PR TITLE
BD-2558: Update list of data included in segment CSV export 

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

Closes #**BD-2558**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/export_braze_data/segment_data_to_csv.md [hunk 1]
```diff
@@ -25,37 +25,65 @@ Due to file size restrictions, your export may fail if the estimated size of you
...
+| gender                      | Gender                                                   |
+| google_ad_ids               | Google advertising IDs associated with the user                      |
+| city                        | City                                     |
+| IDFAs                       | Identifier for Advertising (IDFA) values                 |
+| IDFVs                       | Identifier for Vendor (IDFV) values                      |
+| language                    | Language in ISO-639-1 standard                                        |
+| last_app_version_used       | Last version of the app used                             |
+| last_name                   | Last name                                                |
+| last_session                | Date and time of last session                            |
+| number_of_google_ad_ids     | Count of Google advertising IDs associated               |
```
Comment (rachel-feinberg) [inline _docs/_user_guide/data_and_analytics/export_braze_data/segment_data_to_csv.md:55]: Should "associated" be moved earlier in the description for this field and other similar fields?
```suggestion
| number_of_google_ad_ids     | Count of associated Google advertising IDs               |
```

Comment (bre-fitzgerald) [reply _docs/_user_guide/data_and_analytics/export_braze_data/segment_data_to_csv.md:55]: Sure! Will make that update

File: _docs/_user_guide/data_and_analytics/export_braze_data/segment_data_to_csv.md [hunk 2]
```diff
@@ -25,37 +25,65 @@ Due to file size restrictions, your export may fail if the estimated size of you
...
+| number_of_IDFAs             | Count of IDFAs associated                                |
+| number_of_IDFVs             | Count of IDFVs associated                                |
+| number_of_push_tokens       | Count of push notification tokens associated             |
+| number_of_roku_ad_ids       | Count of Roku advertising IDs associated                 |
+| number_of_windows_ad_ids    | Count of Windows advertising IDs associated              |
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/data_and_analytics/export_braze_data/segment_data_to_csv.md:60]:
```suggestion
| number_of_google_ad_ids     | Count of associated Google advertising IDs               |
| number_of_IDFAs             | Count of associated IDFAs                                |
| number_of_IDFVs             | Count of associated IDFVs                                |
| number_of_push_tokens       | Count of associated push notification tokens             |
| number_of_roku_ad_ids       | Count of associated Roku advertising IDs                 |
| number_of_windows_ad_ids    | Count of associated Windows advertising IDs              |
```

File: _docs/_user_guide/data_and_analytics/export_braze_data/segment_data_to_csv.md [hunk 3]
```diff
@@ -25,37 +25,65 @@ Due to file size restrictions, your export may fail if the estimated size of you
...
+| phone_number                | Phone number                                             |
+| opted_into_push_at          | Date opted into push notifications                       |
+| unsubscribed_from_push_at   | Date unsubscribed from push notifications                |
+| random_bucket               | Random bucket number                                 |
+| roku_ad_ids                 | Roku advertising IDs associated                          |
+| session_count               | Total number of sessions                                 |
+| timezone                    | User's time zone in the same format as the IANA Time Zone Database                                         |
+| in_app_purchase_total       | Total amount spent on in-app purchases                   |
+| user_aliases                | User aliases, if any                                          |
+| windows_ad_ids              | Windows advertising IDs associated                       |
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/data_and_analytics/export_braze_data/segment_data_to_csv.md:70]:
```suggestion
| roku_ad_ids                 | Roku advertising IDs                          |
| session_count               | Total number of sessions                                 |
| timezone                    | User's time zone in the same format as the IANA Time Zone Database                                         |
| in_app_purchase_total       | Total amount spent on in-app purchases                   |
| user_aliases                | User aliases, if any                                          |
| windows_ad_ids              | Windows advertising IDs                       |
```

**Pull request merged**